ui/static/js/Chart.min.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Chart.js
+ * http://chartjs.org/
+ * Version: 2.7.1
```

Comment:
this is not the same file/content - it needs to be the one I patched above

Reply:
Oh, are you using some self-patched secret feature? :P

Reply:
yes, not a secret though, the changes vs stock chart.js are described both in #17 and in the header of the current file

Reply:
Updated